tests/framework/db/CommandTest.php

```diff
@@ -472,7 +472,11 @@ public function testNoTablenameReplacement()
                 'address' => 'Some {{%weird}} address',
             ]
         )->execute();
-        $customerId = $db->getLastInsertID();
+        if ($this->driverName === 'pgsql') {
```

Comment:
How did it work before?

Reply:
It inserted id explicitly but MSSQL doesn't allow inserting of identity columns.

Reply:
How's that connected to pgsql? It was called without arguments before and tests for PostgreSQL passed.

Reply:
No, previously `'id' => 43,` was inserted and then used in select queries. I've removed `id` from insert statement and added `$db->getLastInsertID()`.

Reply:
Ah. OK then.